eng/pipelines/templates/jobs/ci.yml

```diff
@@ -71,6 +71,9 @@ jobs:
       image: $(LINUXVMIMAGE)
       os: linux
 
+    variables:
+      ComponentDetection.SourcePath: '$(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}'
```

Comment:
Does the variable reference within another variable declaration work? I guess if it does, only because these are pre-defined variables.